lib/config/openssh/openssh.go

```diff
@@ -52,11 +63,16 @@ Host *.{{ $clusterName }} {{ $dot.ProxyHost }}
 # Flags for all {{ $clusterName }} hosts except the proxy
 Host *.{{ $clusterName }} !{{ $dot.ProxyHost }}
     Port {{ $dot.Port }}
-    {{- if eq $dot.AppName "tsh" }}
+{{- if eq $dot.AppName "tsh" }}
     ProxyCommand "{{ $dot.ExecutablePath }}" proxy ssh --cluster={{ $clusterName }} --proxy={{ $dot.ProxyHost }}:{{ $dot.ProxyPort }} %r@%h:%p
-{{- end }}{{- if eq $dot.AppName "tbot" }}
+{{- end }}
+{{- if eq $dot.AppName "tbot" }}
+{{- if $dot.PureTBotProxyCommand }}
+    ProxyCommand {{ shellQuote $dot.ExecutablePath }} ssh-proxy-command --destination-dir={{ shellQuote $dot.DestinationDir }} --proxy-server={{ shellQuote (print $dot.ProxyHost ":" $dot.ProxyPort) }} --cluster={{ shellQuote $clusterName }} {{if $dot.TLSRouting }}--tls-routing{{ else }}--no-tls-routing{{ end }} {{if $dot.ConnectionUpgrade }}--connection-upgrade{{ else }}--no-connection-upgrade{{ end }} {{if $dot.Resume }}--resume{{ else }}--no-resume{{end}} --user=%r --host=%h --port=%p
```

Comment:
style nit: unless there's some meaning to having or not having spaces around the expression inside the `{{` `}}`, we should probably be consistent and always have spaces

Reply:
4f6b9a3f76